hadoop-hdds/pom.xml

```diff
@@ -186,6 +193,13 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         <version>${junit.jupiter.version}</version>
         <scope>test</scope>
       </dependency>
+
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>2.2.0</version>
```

Comment:
NIT: can we replace the hard-coded package version?

Reply:
I took this one from `container-service`.  It turns out we have quite a few different version numbers.  So I defined a property for the latest 2.x in root `pom.xml` in 0ebb158fcc793651aa19b3a43bc906fcaa43ae59.

I left Mockito 1.x usage and versions as is, because it's not API-compatible with 2.x.